<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondWindow</class>
 <widget class="QDialog" name="secondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="count_bt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>count</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="firstNum">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>enter first number</string>
   </property>
  </widget>
  <widget class="QLabel" name="SecondNum">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>enter second number</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_lb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>276</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Your result</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_result">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
